addons/auth_signup/i18n/auth_signup.pot

```diff
@@ -390,6 +390,15 @@ msgstr ""
 msgid "Contact"
 msgstr ""
 
+#. module: auth_signup
+#. odoo-python
+#: code:addons/auth_signup/models/res_users.py:0
+#, python-format
+msgid ""
+"Could not contact the mail server, please check your outgoing email server "
+"configuration"
+msgstr ""
+
```

Comment:
No need to add translations manually if we target master.

Reply:
Done